meson.build

```diff
@@ -19,6 +19,11 @@ conf.set('CSP_USE_DEDUP', get_option('use_dedup') == true ? 1 : 0)
 
 conf.set('CSP_HAVE_STDBOOL_H', cc.has_header('stdbool.h') == true ? 1 : 0)
 
+zmq_dep = dependency('libzmq', version: '>4.0', required: false)
+if zmq_dep.found()
```

Comment:
Again, why is this needed here?

We have this

zmq_dep = dependency('libzmq', version: '>4.0', required: false)
if zmq_dep.found()
	csp_sources += files('csp_if_zmqhub.c')
	csp_deps += zmq_dep
endif

Already in sec/interfaces/meson build